sql-plugin/src/main/scala/com/nvidia/spark/rapids/SortUtils.scala

```diff
@@ -201,7 +201,23 @@ class GpuSorter(
           batches.foreach { cb =>
             tabs += GpuColumnVector.from(cb)
           }
-          Table.merge(tabs.toArray, cudfOrdering: _*)
+          // In the current version of cudf merge does not work for structs or lists (nested types)
+          // This should be fixed by https://github.com/rapidsai/cudf/issues/8050
+          val hasNested = {
+            val tab = tabs.head
+            (0 until tab.getNumberOfColumns).exists { i =>
+              tab.getColumn(i).getType.isNestedType
+            }
+          }
+          if (hasNested) {
+            // so as a work around we concat all of the data together and then sort it.
+            // It is slower, but it works
+            withResource(Table.concatenate(tabs: _*)) { concated =>
```

Comment:
nit: spelling concatenate

Reply:
I don't know what you mean.  Do you want me to fully spell out "concatenate" instead of "concat" or "concatenated" instead of my shortened "concated"?

Reply:
Should we file a follow-on in the plugin for us to revisit this when the cudf issue is done?

Reply:
if you meant for this to be then I see no problem